clipper_admin_v2/clipper_admin/clipper_admin.py

```diff
@@ -0,0 +1,488 @@
+from __future__ import absolute_import, division, print_function
+import logging
+import docker
+import requests
+import json
+import os
+from .container_manager import CONTAINERLESS_MODEL_IMAGE
+
+DEFAULT_LABEL = ["DEFAULT"]
+
+logger = logging.getLogger(__name__)
+
+
+class ClipperException(Exception):
+    pass
+
+
+def start_clipper(cm):
```

Comment:
You're right, that does seem weird. At first I was thinking that plain functions would be cleaner and you could potentially wrap this in a CLI, but given that there's still this container management state that needs to be tracked, we should create a class and track it for the user.